src/Component/SelectMenu/ChannelSelectMenu.php

```diff
@@ -32,9 +32,6 @@ public function __construct(
 
     public function get(): array
     {
-        return array_merge(
-            parent::get(),
-            is_null($this->channelTypes) ? [] : ['channel_types' => $this->channelTypes]
-        );
+        return [...parent::get(), ...is_null($this->channelTypes) ? [] : ['channel_types' => $this->channelTypes]];
```

Comment:
Is there any benefit to this at all other than alternative syntax?

Reply:
Sure, _massive_ performance benefit. See: https://onlinephp.io/c/ab317